pkg/compactor/shuffle_sharding_grouper.go

```diff
@@ -109,6 +115,7 @@ func (g *ShuffleShardingGrouper) Groups(blocks map[ulid.ULID]*metadata.Meta) (re
 			// TODO: Use the group's hash to determine whether a compactor should be responsible for compacting that group
 			groupHash := hashGroup(group.blocks[0].Thanos.Labels["__org_id__"], group.rangeStart, group.rangeEnd)
 
+			g.remainingPlannedCompactions.Inc()
```

Comment:
If the metric is scraped in the middle of this function, it looks like it will have some value between 0 and the final value.  Is that useful?
Would it be better to count up then `Set()` once?